packages/progress-indicator/src/LionProgressIndicator.js

```diff
@@ -78,12 +130,64 @@ export class LionProgressIndicator extends LocalizeMixin(LitElement) {
 
   connectedCallback() {
     super.connectedCallback();
-    this.setAttribute('role', 'status');
-    this.setAttribute('aria-live', 'polite');
+    this.setAttribute('role', 'progressbar');
+  }
+
+  /**
+   * Update aria labels on state change.
+   * @param {import('@lion/core').PropertyValues } changedProperties
+   */
+  updated(changedProperties) {
+    super.updated(changedProperties);
+
+    if (this.indeterminate) {
+      if (changedProperties.has('_ariaLabel') || changedProperties.has('_ariaLabelledby')) {
+        this._setDefaultLabel();
+      }
+    } else {
+      if (changedProperties.has('value')) {
```

Comment:
This can also mean value just became undefined, right? And then it would be indeterminate and the checks below would fail. 

It works of course since indeterminate is a static getter that reads the value, but it's hard to read/comprehend 😅 
And would most likely break if we would update status of indeterminate asynchronously (c.q. inside the update cycle).

Reply:
You prefer to change it to:

```js
if (changedProperties.has('value') && typeof this.value === 'number') {
```